src/src.pro

```diff
@@ -62,6 +62,7 @@ msvc:QMAKE_CXXFLAGS += -MP
 macx:QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.7
 
 QT += network opengl uitools multimedia gui
+qtHaveModule(gamepad): QT += gamepad
```

Comment:
Do we not need to import the same sort of thing into the CMake project files/build system?